src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx

```diff
@@ -57,22 +68,26 @@ export const FairHeaderImage: React.FC<FairHeaderImageProps> = ({
       </Media>
 
       <Media greaterThan="xs">
-        <ResponsiveBox
+        <Container
           position="relative"
           mx="auto"
-          aspectWidth={image._1x.width}
-          aspectHeight={image._1x.height}
-          maxWidth={375}
+          maxWidth="100%"
           bg="black10"
+          width={`${WIDTH_VH}vh`}
+          height={`${HEIGHT_VH}vh`}
         >
-          <Image
-            src={image._1x.src}
-            srcSet={`${image._1x.src} 1x, ${image._2x.src} 2x`}
-            alt={name}
-          />
+          <picture>
+            <source
+              srcSet={`${image.lg_1x.src} 1x, ${image.lg_2x.src} 2x`}
+              media="(min-height: 900px)"
+            />
 
+            <source srcSet={`${image.md_1x.src} 1x, ${image.md_2x.src} 2x`} />
+
+            <Image src={image.md_1x.src} alt={name} lazyLoad />
+          </picture>
```

Comment:
Picture tag here so as to swap between md/lg versions of the image according to the height of the viewport.